e2e/events.spec.js

```diff
@@ -0,0 +1,86 @@
+const { takeScreenshot } = require("./helpers");
+const { formatDateRange } = require("../src/data/formatters");
+
+const formattedDateNumber = num => (num < 10 ? `0${num}` : num);
+
+const today = new Date();
```

Comment:
I don't love this date logic here, but I can't work out a better way to test the filtering that selecting today through the end of the month... any ideas?

Reply:
I think that should be fine.